modules/nodes-nodes-rebooting-router.adoc

```diff
@@ -3,7 +3,7 @@
 // * nodes/nodes-nodes-rebooting.adoc
 
 [id="nodes-nodes-rebooting-router_{context}"]
-= Understanding how to reboot nodes running routers
+= About rebooting a node that is running a router
```

Comment:
Is this module really an ops thing that cluster admins should care about?  It seems more like "if you are creating your own router load balancer, here's a bit of context".  Which would be an install-time thing.  As it stands, it's not clear to me who reads this module, or what action we have enabled them to take post-reading.